### DISCUSSION TITLE
Giscus support

### DISCUSSION THREAD

**Discussion by marcosps** [adityatelange/hugo-PaperMod#1193] [Ideas]
It seems that Giscus is a good comments system. Is there a plan to have by default on papermod, and only enable it by using some configs?

Thanks in advance!

**Comment by ventz** [comment]
Just came to ask the same

**Reply by marcosps** [reply]
For the record, I went and added it myself, fixed in the theme, as I was going to use it anyway :) You should do the same, it's not difficult at all, since there isn't support for it yet.

**Reply by ventz** [reply]
@marcosps Thanks.

From what I can see, someone documented it here: https://main--cheerful-mousse-b9d87b.netlify.app/posts/setup_site/#comments

I was really hoping for an official add, as I am trying to avoid "one off" changes, but given that this issue has been open for almost a year, I am assuming there is no interest? (A bit surprised as it seems most are using giscus).

**Reply by marcosps** [reply]
I agreed. While there isn't official support, the solution is fix it ourselves. I didn't send a PR because I'm not a web developer, so I'm not exactly sure how to setup parameters to enable it or not for other users...

**Reply by ventz** [reply]
@marcosps It seems incredibly easy to do this. Wanted to post here because I figured out how to change the giscus theme linked to the papermod theme changer and thought you may want it too:

https://github.com/giscus/giscus/issues/336#issuecomment-1927613713

Specifically, note the working `layouts/partials/comments.html` section I pasted.

**Reply by marcosps** [reply]
Thanks a lot @ventz! I would love to have it builtin on PaperMod, since I'm a terrible web developer and I hate changing the default theme to do such things :P Thanks for creating such an easy step-by-step!

**Comment by adityatelange** [comment]
Hope this helps https://github.com/adityatelange/hugo-PaperMod/discussions/1445

**Reply by tgagor** [reply]
I'm looking for this discussion - it's gone now. Did you share it also anywhere else?

**Reply by adityatelange** [reply]
It was useless, deleted.

**Reply by tgagor** [reply]
Then it have to stay black :)
Thanks for fast response.
